lms/static/js/commerce/credit.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Credit-related utilities
+ */
+var edx = edx || {};
+
+(function ($) {
+    'use strict';
+
+    edx.commerce = edx.commerce || {};
+    edx.commerce.credit = edx.commerce.credit || {};
+
+    edx.commerce.credit.createCreditRequest = function (providerId, courseKey, username, error) {
+        $.ajax({
```

Comment:
I recommend returning the result of ``$.ajax`` which is a promise that callers can use. You then don't need to have the optional error parameter, and it also allows the caller to chain other ``done`` functions together. We try to always use promises over callbacks to avoid getting into callback hell.